pages/getting-started/writing-your-first-test.md

```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
+Parece ser fácil, vamos ver qual deles gostamos... que tal `type`?
+
+Para encontrar esse elemento pelo seu conteúdo, usamos `cy.contains()`.
+
+Vamos adicionar isso em nosso teste e ver o que acontece:
+
+```javascript
+describe('My First Test', () => {
+  it('finds the content "type"', () => {
+    cy.visit('https://example.cypress.io')
+
+    cy.contains('type')
+  })
+})
+```
+
+Agora o teste deve exibir o comando `CONTAINS` e ainda estar verde.
+
+Mesmo sem adicionar uma asserção, sabemos que está tudo certo! Isso acontece pois muitos dos comandos
+do Cypress são construídos para falhar caso não encontram o que é esperado. Isso é conhecido como uma
+[Asserção Padrão](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Default-Assertions).
+
+Para confirmar, substitua `type` por qualquer palavra que não esteja na página, como `hype`.
+Perceba que o teste falha, mas só depois de 4 segundos!
+
+Entende o que o Cypress está fazendo? Automaticamente espera e tenta novamente pois é esperado que
+eventualmente o elemento será encontrado no DOM. O teste não falha imediatamente.
+
+![Teste com falha](https://docs.cypress.io/img/guides/first-test-failing-contains.529e740a.png)
+
+```markdown
+Mensagens de Erro
+
+Tomamos o cuidado de escrever centenas de mensagens de erro personalizadas que tentam explicar claramente
+o que deu errado no Cypress. Nesse caso, o Cypress atingiu o tempo limite tentando encontrar o texto hype
+em toda a página. Para ler mais sobre a exibição do erro, leia sobre
+[Erros de depuração](https://docs.cypress.io/guides/guides/debugging.html#Errors).
+```
+
+Antes de adicionar outro comando - vamos consertar esse teste. Substitua `hype` por `type`.
+
+[Vídeo de exemplo do teste acima](https://docs.cypress.io/img/snippets/first-test-contains-30fps.mp4)
+
+### Passo 3: Clicar em um elemento
+
+Ok, agora queremos clicar no link que encontramos. Como fazer isso? Adicione o comando `.click()`
+no final do comando anterior, como abaixo:
+
+```javascript
+describe('My First Test', () => {
+  it('clicks the link "type"', () => {
+    cy.visit('https://example.cypress.io')
+
+    cy.contains('type').click()
+  })
+})
+```
+
+É quase possível ler o teste como uma história! Cypress chama isso de encadeamento e comandos são
+encadeados para contruir testes que realmente expressam o que a aplicação faz de forma declarativa.
+
+Note também que o painel da aplicação foi atualizado depois do clique, abrindo o link e mostrando a
+página de destino.
+
+Agora podemos fazer uma asserção nessa nova página!
+
+[Vídeo de exemplo do teste acima](https://docs.cypress.io/img/snippets/first-test-click-30fps.mp4)
+
+```markdown
+Você pode ver o IntelliSense em seus arquivos de teste ao adicionar uma única linha de comentário especial.
+Leia sobre isso [aqui](https://docs.cypress.io/guides/tooling/IDE-integration.html#Triple-slash-directives).
+```
+
+### Passo 4: Fazer uma asserção
+
+WIP
```

Comment:
I think we can remove this for now and add the HTML code for back to top as we see in the other pages.

Reply:
Fixed